ncm-metaconfig/src/main/perl/metaconfig.pm

```diff
@@ -44,16 +42,6 @@ sub prepare_action
         }
     }
 
-    if ($srv->{daemon}) {
```

Comment:
yes, it has happened before and might/will happen again (we have quiet a few self-packaged parts of quattor with master branch + PRs merged in (or branches under development), and not always sync back the schema of each new package with the actual schema).
it's not good practise, but quattor does not protect against this. enforcing the schema with the component should be possible (and is probably quite easy to do).
it's actually worse then that considering all non-locked quattor parts like CAF and CCM (at least we don't lock them). our packages have no version dependencies, and there are quite some things that won't work without aligned versions. (e.g. imagine your repo sync failed and you skipped CCM or CAF; and you deploy 15.2).

anyway, deprecation should done with care :smile: